src/rubicon/objc/eventloop.py

```diff
@@ -17,14 +18,34 @@
 from .types import CFIndex
 
 if sys.version_info < (3, 14):
-    from asyncio import SafeChildWatcher
+    from asyncio import (
+        AbstractEventLoopPolicy,
+        DefaultEventLoopPolicy,
+        SafeChildWatcher,
+        new_event_loop,
+        set_event_loop_policy,
+    )
+else:
+    # Python 3.14 finalized the deprecation of SafeChildWatcher. There's no
+    # replacement API; the feature can be removed.
+    #
+    # Python 3.14 also started the deprecation of event loop policies, to be
+    # finalized in Python 3.16; there was some symbol renaming to assist in
+    # making the deprecation visible. See
+    # https://github.com/python/cpython/issues/127949 for details.
+    from asyncio import (
+        _AbstractEventLoopPolicy as AbstractEventLoopPolicy,
+        _DefaultEventLoopPolicy as DefaultEventLoopPolicy,
+    )
```

Comment:
I assume those imports will stop working with Python 3.16. Is there anything we can do to prepare for this now in addition to the deprecation warning below?

Reply:
Sure - since we know the deprecation will be finalised with 3.16, we can make these imports and the actual EventLoopPolicy definition conditional on `version_info < (3, 16)`. That way, when, in 6 years time, we run the `pyupgrade` with the 3.16 minimum version, all the dead code will be automatically removed.

Reply:
Thanks! No strong opinions here, I'll leave it to you whether you think thats worth doing :)